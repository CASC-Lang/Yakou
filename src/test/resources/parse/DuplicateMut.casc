mut mut class DuplicateMut

impl DuplicateMut {
    mut mut fn a1(self) {}
}