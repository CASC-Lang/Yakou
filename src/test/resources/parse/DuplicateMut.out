error: Duplicate `mut` keyword
--> ./DuplicateMut.casc:1:0
1 | mut mut class DuplicateMut
    ^^^= hint: Encountered first one here
2 |
error: Duplicate `mut` keyword
--> ./DuplicateMut.casc:1:4
1 | mut mut class DuplicateMut
        ^^^= hint: Duplicate here
2 |
error: Duplicate `mut` keyword
--> ./DuplicateMut.casc:4:4
3 | impl DuplicateMut {
4 |     mut mut fn a1(self) {}
        ^^^= hint: Encountered first one here
5 | }
error: Duplicate `mut` keyword
--> ./DuplicateMut.casc:4:8
3 | impl DuplicateMut {
4 |     mut mut fn a1(self) {}
            ^^^= hint: Duplicate here
5 | }